src/Dapr.Client/DaprClient.cs

```diff
@@ -783,6 +783,76 @@ public async Task<StateEntry<TValue>> GetStateEntryAsync<TValue>(string storeNam
             StateOptions stateOptions = default,
             IReadOnlyDictionary<string, string> metadata = default,
             CancellationToken cancellationToken = default);
+        /// <summary>
+        /// Define a SaveStateAsync task that gets a binary input   
+        /// </summary>
+        /// <param name="storeName">The  </param>
+        /// <param name="key"></param>
+        /// <param name="binaryValue"></param>
+        /// <param name="stateOptions"></param>
+        /// <param name="metadata"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public abstract Task SaveByteStateAsync(
```

Comment:
I'd go with the term _Raw_ over _Byte_ and put it after _State_. So `SaveStateRawAsync`, to make it line up with `GetBulkStateRawAsync` which is already [in master](https://github.com/dapr/dotnet-sdk/blob/master/src/Dapr.Client/DaprClientGrpc.cs#L642)

Reply:
Thanks for the review. named these methods similar to PublishByteEventAsync which is also in master. Hope that works. Thank you.

Reply:
Oh wow, I missed that. So the SDK already inconsistent in this respect. 😑
Tbh I'm really hoping this PR gets approved soon so we can skip serialization/deserialization when working with blob storage.